docs/contents/cloud-datastream.json

```diff
@@ -9,11 +9,7 @@
         {
             "title": "DatastreamClient (v1)",
             "type": "datastream\/v1\/datastreamclient"
-        },
```

Comment:
How/why were these in here before the client was added?

Reply:
I added these files as a follow up to a batch of new libraries that were released a few months back, held off on datastream due to the missing location proto classes, but must have missed removing this at the time. Shouldn't have an effect on anything.